src/azure-cli/azure/cli/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/__init__.py

```diff
@@ -0,0 +1,30 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.command_modules.netappfiles._help import helps  # pylint: disable=unused-import
+
+
+class NetAppFilesCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        netappfiles_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.netappfiles.custom#{}')
+        super(NetAppFilesCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                        custom_command_type=netappfiles_custom)
```

Comment:
You need to register a resource type for this in the core profile and reference it here. @limingu for guidance.

Reply:
@leonardbf, to register the netappfiles resource, you need to add one new ResourceType after https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L87 and add resource_type=ResourceType.MGMT_NETAPPFILES, here.

Reply:
ok. Done.